app/controllers/api/v1x0/portfolio_items_controller.rb

```diff
@@ -36,16 +40,27 @@ def destroy
         head :no_content
       end
 
+      def copy
+        svc = Catalog::CopyPortfolioItem.new(portfolio_copy_params)
+        render :json => svc.process.new_portfolio_item
+      rescue ActiveRecord::RecordNotFound => e
+        json_response({ :errors => e.message }, :unprocessable_entity)
+      end
+
       private
 
       def portfolio_item_params
-        params.require(:service_offering_ref)
```

Comment:
This require isn't needed anymore?

Reply:
No - it's being required in the `before_action`.